foreman.te

```diff
@@ -448,10 +454,12 @@ corenet_tcp_bind_vnc_port(websockify_t)
 corenet_tcp_bind_vnc_port(foreman_rails_t)
 dev_read_urand(websockify_t)
 kernel_read_system_state(websockify_t)
-logging_send_syslog_msg(websockify_t)
+optional_policy(`
+    logging_send_syslog_msg(websockify_t)
+    abrt_stream_connect(websockify_t)
```

Comment:
These modules are independent so should be in separate optional blocks.

Reply:
Good find, fixed. Amended.